Remotion/Web/Development.WebTesting/WebTestSetUpFixtureHelper.cs

```diff
@@ -107,6 +117,51 @@ private void HostWebApplication ()
       _hostingStrategy.DeployAndStartWebApplication();
     }
 
+    private void VerifyWebApplicationStarted (string webApplicationRoot, TimeSpan applicationPingTimeout)
+    {
+      var stopwatch = Stopwatch.StartNew();
+      
+      var webRequest = (HttpWebRequest)HttpWebRequest.Create (webApplicationRoot);
+      webRequest.Method = WebRequestMethods.Http.Head;
+      webRequest.AllowAutoRedirect = true;
+      
+      //Just setting this to some value for the first run
+      var statusCode = HttpStatusCode.NotFound;
+
+      while (statusCode != HttpStatusCode.OK)
+      {
+        try
+        {
+          var response = (HttpWebResponse) webRequest.GetResponse();
+          statusCode = response.StatusCode;
+
+          if (stopwatch.ElapsedMilliseconds > applicationPingTimeout.TotalMilliseconds)
+          {
+            throw new InvalidOperationException (
+                $"Checking the Web Application Root '{webApplicationRoot}' did not return '{HttpStatusCode.OK}' in the defined {nameof (applicationPingTimeout)} '{applicationPingTimeout}'."
+                + $"Last statusCode: '{statusCode}'. Abort.");
+          }
+          else
+          {
+            s_log.Warn ($"Checking the Web Application Root '{webApplicationRoot}' failed with Status Code '{statusCode}'. Trying again");

```

Comment:
I think all log messages need periods too if I'm not mistaken
```suggestion
            s_log.Warn ($"Checking the Web Application Root '{webApplicationRoot}' failed with Status Code '{statusCode}'. Trying again.");
```